readme.md

```diff
@@ -1,51 +1,153 @@
-# Ewon Connector
+
+# Ignition Ewon Connector
+
+## [Table of Contents](#table-of-contents)
+
+1. [Description](#description)
+2. [Version](#version)
+3. [Support](#support)
+4. [Features](#features)
+5. [Download](#download)
+6. [Installation](#installation)
+7. [Setup](#setup)
+8. [Connector Usage](#connector-usage)
+
+
+## [Description](#table-of-contents)
 
 Synchronize Ewon data to Ignition's Tag Historian
 
-## Version
+## [Version](#table-of-contents)
 
 This module version is compatible with Ignition 8.  For an Ignition 7.9 compatible module go to [Ignition-7.9-master](https://github.com/hms-networks/eWonConnector/tree/Ignition-7.9-master) branch.
 
-## Download
+## [Support](#table-of-contents)
 
-The module can be downloaded from the [releases page](https://github.com/hms-networks/eWonConnector/releases).
+This application is supported by HMS' North American offices.
+
+| Phone                                        | Forum                                      | Email                                           |
+|:--------------------------------------------:|:------------------------------------------:|:-----------------------------------------------:|
+| [+1 312 829 0601](tel:13128290601), Option 2 | [hms.how](https://forum.hms-networks.com/) | [support@hms.how](mailto:support@hms.how)       |
 
-## Features
+## [Features](#table-of-contents)
 
-* Data synchronization between Ewon DataMail and Ignition
-   * Tag value updated up to once per minute
+* Data synchronization between Ewon DataMailbox and Ignition
```

Comment:
I think it's better without "DataMailbox". Synchronization can be done with DataMailbox, but it can also be done with M2Web.

Also, I think the references to DataMailbox/Talk2M here are confusing. From the Ewon developer website, I've always gotten the idea that Talk2M is the overarching system, and DataMailbox and M2Web both fall under Talk2M, https://developer.ewon.biz/registration

Reply:
For this specific line i think its important to be clear that this is using datamailbox.


I specifically never mention m2web

Reply:
Datamailbox is what is providing the historical / offline data functionality.

Reply:
I should've put in the original comment, but that comment is referring to the section as a whole, not the specific line.

I just think that in general, this section is a little bit confusing. Something along these lines reads better imo,

- Read Functionality
   - Historical (via DataMailbox) (default option)
      - Updated up to once per minute
   - Realtime (via M2Web)
      - Updated up to once per second
      - Can be enabled per-tag, per-Ewon or system-wide
- Write Functionality
   - Modify Ewon tags from Ignition